helm/etcd-backup-operator/templates/cronjob.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: batch/v2alpha1
+apiVersion: batch/v1
```

Comment:
IIUC today, this should go to `v2beta1` instead?

Reply:
Yep there is another PR that move it to v2beta1 instead . This PR is blocked until a later date when v1beta1 is deprecated.